community/syslog-summary/APKBUILD

```diff
@@ -0,0 +1,43 @@
+# Contributor: Stuart Cardall <developer@it-offshore.co.uk>
+# Maintainer: Stuart Cardall <developer@it-offshore.co.uk>
+pkgname=syslog-summary
+pkgver=1.14
+pkgrel=0
+pkgdesc="Summarize the contents of a syslog log file / useful with logcheck"
+url="https://packages.debian.org/sid/syslog-summary"
```

Comment:
This doesn’t look like the upstream, it’s a package for https://github.com/dpaleino/syslog-summary. Why do you download it from Debian instead of the upstream repository?